crates/ruff_linter/src/rules/pylint/rules/collapsible_else_if.rs

```diff
@@ -122,15 +126,30 @@ fn convert_to_elif(
         stylist,
     )?;
 
+    // If there's trivia, restore it
+    let trivia = if trivia_range.is_empty() {
+        None
+    } else {
```

Comment:
Is `trivia_range` ever going to be empty? Because there should at least be an indentation between the `else:` line end and the start of the first statement.

Reply:
Good question.. uhh let me look.

Reply:
I put a debug statement in there and the trivia range is indeed empty when there's no trivia 🤷‍♀️